github/ci/prow-deploy/kustom/base/configs/current/config/config.yaml

```diff
@@ -420,6 +420,17 @@ presets:
   volumeMounts:
   - name: docker-graph
     mountPath: /docker-graph
+- labels:
+    preset-pinc-enabled: "true"
```

Comment:
I think you are right here - it might be better to use the full name so it easier to understand. I will make that update now.

Reply:
I think pinc is known a lil
https://github.com/kukaryambik/pinc
imho short name is better

Reply:
@oshoval I disagree. I think we rather should be clear here and not force the reader to have to remember what this preset is about.

And since Brian has made the change already, let's go with it.

Reply:
Bttw I think that at the end pinc will come a term like dind, but whatever you prefer